facts/4.2/centos-9-x86_64.facts

```diff
@@ -1,4 +1,5 @@
 {
+  "selinux": false,
```

Comment:
I am sorry, I should have looked more closely at the patch earlier: In the existing CentOS-9 and OracleLinux-9 files, SELinux is enabled, so this is an inconsistency.